.github/workflows/publish.yml

```diff
@@ -2,24 +2,34 @@ name: Publish
 on:
   push:
     tags:
-    - 'v0.4.*'
+    - 'v0.5.*'
+  workflow_dispatch
+  schedule:
+  # Snapshot release every second work day
+  - cron: '0 0 * * 1,3,5'
 
 jobs:
   test-linux:
+    if: github.event_name != 'schedule' && github.repository == 'scala-native/scala-native'
     uses: ./.github/workflows/run-tests-linux.yml
   test-windows:
+    if: github.event_name != 'schedule' && github.repository == 'scala-native/scala-native'
     uses: ./.github/workflows/run-tests-windows.yml
   test-macos:
+    if: github.event_name != 'schedule' && github.repository == 'scala-native/scala-native'
     uses: ./.github/workflows/run-tests-macos.yml
   test-jdk-compliance:
+    if: github.event_name != 'schedule' && github.repository == 'scala-native/scala-native'
     uses: ./.github/workflows/run-jdk-compliance-tests.yml
   test-multiarch:
+    if: github.event_name != 'schedule' && github.repository == 'scala-native/scala-native'
     uses: ./.github/workflows/run-tests-linux-multiarch.yml
 
   publish:
     name: Publish
     runs-on: ubuntu-22.04
     needs: [test-linux, test-windows, test-macos, test-jdk-compliance, test-multiarch]
+    if: github.repository == 'scala-native/scala-native'
```

Comment:
I don't think this is working? The `publish` job is skipped because it needs the `test-*` jobs, but those are skipped if the workflow runs on schedule.

https://github.com/scala-native/scala-native/actions/runs/5593698843

Reply:
Yes, you're correct. I really thought I fixed it already. Oh well, I will fix it soon

Reply:
Oh, right, it's already on main: https://github.com/scala-native/scala-native/commit/61f4230a0bb7fe6c0fc58496ffddf4fd319debf0